pom.xml

```diff
@@ -126,13 +127,13 @@
             <configuration>
               <target>
                 <get dest="src/main/resources" skipexisting="true">
-                  <url url="https://registry.npmjs.org/@esbuild/darwin-arm64/-/darwin-arm64-${esbuild.version}.tgz"/>
```

Comment:
is there no other way to build this than depend on a personal github account releases ? or is this just a draft/experiment?

Reply:
Probably we should move that to the mvnpm namespace

Reply:
Yes, that is the plan, until (if ever) it is supported in esbuild. @edewit did you do a PR against esbuild ? Or are they not interested in this ?

Reply:
I have @phillip-kruger added a PR that will introduce an API to be able to add plugins to the cli, but we will still need to build or own version with the plugins that we want to use.